runtime/flangrti/CMakeLists.txt

```diff
@@ -66,7 +67,11 @@ add_flang_library(flangrti_static
   ${PGC_SRC_FILES}
   ${SHARED_SOURCES}
   )
-set_property(TARGET flangrti_static PROPERTY OUTPUT_NAME flangrti)
+if (MSVC)
```

Comment:
Same as above

Reply:
Can we teach the driver to look for `flangrti.dll` instead of `libflangrti.dll`?

Reply:
In MSVC, the structure of the files is like
- flang.lib  - import library for dynamic library
- flang.dll  - dynamic library
- libflang.lib - static library

Notice that the import library and the static library have the same extension and hence the need for two different names.

For MinGW,
- libflang.dll.a - import library for dynamic library
- libflang.dll - dynamic library
- libflang.a  - static library

Reply:
I see, life is complicated. Thanks.